configs/recognition_audio/resnet/tsn_resnet_r18_64x1x1_100e_kinetics400_audio_feature.py

```diff
@@ -90,7 +90,7 @@
 # runtime settings
 dist_params = dict(backend='nccl')
 log_level = 'INFO'
-work_dir = './work_dirs/tsn_r18_64x1x1_100e_kinetics400_audio_feature/'
+work_dir = './work_dirs/tsn_resnet_r18_64x1x1_100e_kinetics400_audio_feature/'
```

Comment:
why indicating `resnet_r18` not just `r18`

Reply:
Right. I have re-organized this one. `resnet` and `resnet_audio` denote two different backbones. Now it is called `resnet` and `audioonly` (a similar way as slowonly).